debian/NOTES.md

```diff
@@ -0,0 +1,112 @@
+# NOTES
+
+used ubuntu 20.04 system
+
+## Pre Requisites
+
+```shell
+$ rustup upgrade
+$ sudo apt-get update && sudo apt-get install -y cmake g++ libprotobuf-dev protobuf-compiler
+$ cargo install cargo-deb
+```
+
+## Optinal test build SurrealDB
+
+```shell
+$ make build
+```
+
+## Make Debian
+
+```shell
+$ make debian
+# or 
+$ cargo deb
+# outcome
+...
+info: Generating maintainer script postrm
+info: compressed/original ratio 6274596/20791808 (30%)
+target/debian/surreal_1.0.0~beta.5_amd64.deb
+```
+
+## Install local deb
+
+```shell
+$ sudo dpkg -i target/debian/surreal_1.0.0~beta.5_amd64.deb 
+```
+
+## Some Service 
```

Comment:
Just confiming, this `surreal` service is installed with the deb package yeah?

Reply:
Yes everything work out of the box just ibstall and done